pressian.com.js

```diff
@@ -0,0 +1,32 @@
+{
+	"translatorID": "3f7b7b18-1152-40b6-b9e4-e397f913641b",
+	"label": "pressian.com",
+	"creator": "mete0r@sarangbang.or.kr",
+	"target": "^https?://www.pressian.com/",
```

Comment:
Escape periods. Also, based on your detection code below (`detect`), you should just include that part in the target regexp. In Scaffold, it should be `^https:?//www\.pressian\.com/news/article\.html`

The `detect` part can then just be `function() { return true; }` (I think)